hi-ml-multimodal/setup.py

```diff
@@ -3,85 +3,36 @@
-# https://github.com/pypa/gh-action-pypi-publish
-GITHUB_REF_TAG_COMMIT = 'refs/tags/v'
-
-github_ref = os.getenv('GITHUB_REF')
-if github_ref and github_ref.startswith(GITHUB_REF_TAG_COMMIT):
-    version = github_ref[len(GITHUB_REF_TAG_COMMIT):]
-
-# Otherwise, if running from a GitHub Action, but not a tagged commit then GITHUB_RUN_NUMBER will be populated.
-# Use this as a post release number. For example if GITHUB_RUN_NUMBER = 124 then the version string will be
-# '99.99.post124'. Although this is discouraged, see:
-# https://www.python.org/dev/peps/pep-0440/#post-releases
-# it is necessary here to avoid duplicate packages in Test.PyPI.
-if not version:
-    build_number = os.getenv('GITHUB_RUN_NUMBER')
-    if build_number:
-        # In github workflows, we may pull in hi-ml-multimodal as a dependency. Usually, we have a condition like
-        # hi-ml-multimodal>=0.1.5. This means that a package version from PyPi would trump the local wheels. For this
-        # reason, use an extremely large version number to give the local wheel priority.
-        version = '99.99.post' + build_number
-    else:
-        default_random_version_number = floor(random() * 10_000_000_000)
-        version = f'99.99.post{str(default_random_version_number)}'
-
-package_name = 'hi-ml-multimodal'
-(here / 'package_name.txt').write_text(package_name)
-(here / 'latest_version.txt').write_text(version)
-
-# Read run_requirements.txt to get install_requires
-install_requires = (here / 'requirements_run.txt').read_text().split("\n")
-# Remove any whitespace and blank lines
-install_requires = [line.strip() for line in install_requires if line.strip()]
-
-description = 'Microsoft Health Futures package to work with multi-modal health data'
+description = "Microsoft Health Futures package to work with multi-modal health data"
 
 setup(
     name=package_name,
     version=version,
     description=description,
     long_description=long_description,
-    long_description_content_type='text/markdown',
-    url='https://github.com/microsoft/hi-ml',
-    author="Microsoft Research Cambridge Medical Imaging Team ",
+    long_description_content_type="text/markdown",
+    url="https://github.com/microsoft/hi-ml",
+    author="Biomedical Imaging Team @ Microsoft Health Futures",
     author_email="innereyedev@microsoft.com",
     classifiers=[
-        'Development Status :: 3 - Alpha',
-        'Intended Audience :: Science/Research',
+        "Development Status :: 3 - Alpha",
+        "Intended Audience :: Science/Research",
         "Topic :: Scientific/Engineering :: Medical Science Apps.",
-        'License :: OSI Approved :: MIT License',
-        'Programming Language :: Python :: 3.7'
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python :: 3.7"
     ],
-    keywords='HealthIntelligence',
-    license='MIT License',
+    keywords="HealthIntelligence",
```

Comment:
Some suggestions for more keywords we could add: Health Futures, Multimodal

Reply:
I've just removed this. Probably not very useful anyway.